src/main/java/com/academia/andruhovich/library/model/Author.java

```diff
@@ -0,0 +1,30 @@
+package com.academia.andruhovich.library.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
```

Comment:
[Let's a bit change annotations' order:
@Data
@NoArgsConstructor
@AllArgsConstructor 
@Entity
@Table(name = "author")  keep this for all entities](url)